test/provider.test.tsx

```diff
@@ -93,7 +86,7 @@ describe('react provider', () => {
 
     app.find('button').simulate('click');
 
-    setTimeout(() => expect(app.find('.field').text()).toBe('field=1'));
+    setTimeout(() => expect(app.find('.field').text()).toBe('field=1'), 200);
```

Comment:
😂

Reply:
This is a try.  But it is not useful. The listener of field changes is currently not covered by the test.